meson.build

```diff
@@ -59,6 +59,12 @@ else
   cc_pam_flags = []
 endif
 
+if get_option('sched')
```

Comment:
does this need to be an option ?  `sched.h` is in POSIX:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sched.h.html